Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/SearchParamters.java

```diff
@@ -3,6 +3,6 @@
 public interface SearchParamters {
 
 	public final static String OPERATOR = "operator";
-	public final static String OP_AND = "and";
-	public final static String OP_OR = "or";
+    public final static String OP_AND = "AND";
```

Comment:
@jdamerow These operators should be in caps. Else we cant get the exact result that is required. Shall I change this now? Or can I make the test cases fail and these changes will be merged from develop.